tracer/test/test-applications/integrations/Samples.WeakHashing/Program.cs

```diff
@@ -55,6 +60,7 @@ private static void testHashAlgorithm(HashAlgorithm algorithm)
 #if NET5_0_OR_GREATER
             _ = algorithm.ComputeHashAsync(stream, CancellationToken.None).Result;
 #endif
+            System.Threading.Thread.Sleep(100);
```

Comment:
Assuming this was for debugging purposes and can be removed?

Reply:
I found that in the integration test, for some reason, the spans could rarely be sent in a different order, which would cause flakiness, so this helps to avoid that

Reply:
We should investigate this issue deeply. Sleeping and having timing dependencies is a source for flakiness.